source-mongodb/.snapshots/TestPopulateDatabaseResumeTokens-captures_new_documents

```diff
@@ -1,20 +0,0 @@
-# ================================
```

Comment:
The various `TestPopulateDatabaseResumeTokens-*` snapshots were used to test a previous migration, for which the code no longer exists and the snapshots are no longer needed.